app/services/post_webhook.rb

```diff
@@ -0,0 +1,72 @@
+require 'base64'
+require 'rest-client'
+
+class PostWebhook
+  USER_AGENT = 'Redhat Insights Portal v1.0.0'.freeze
+  SIGNATURE_KEY = 'X-Service-Portal-Signature'.freeze
+
+  def initialize(options)
+    @options = options
+    validate
+    default_headers
```

Comment:
Minor but the method default_headers probably should just return the headers, and the assignment to `@headers` happen here.

Reply:
Either that or name the method as a verb, so `set_default_headers`.  Then, the method `authorization` should also be a verb such as `add_authorization_headers